<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SslInfo</class>
 <widget class="QWidget" name="SslInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>510</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="encryptionIndicator">
     <property name="text">
      <string>[padlock]</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="explanation">
     <property name="text">
      <string comment="KDE::DoNotExtract">Bruce Schneier secure</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="addressTag">
     <property name="text">
      <string comment="Web page address">Address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KSqueezedTextLabel" name="address">
     <property name="text">
      <string comment="KDE::DoNotExtract">KSqueezedTextLabel</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ipTag">
     <property name="text">
      <string>IP address:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ip">
     <property name="text">
      <string comment="KDE::DoNotExtract">is not there</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="encryptionTag">
     <property name="text">
      <string>Encryption:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="encryption">
     <property name="text">
      <string comment="KDE::DoNotExtract">SnakeOilCrypt 3000</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="detailsTag">
     <property name="text">
      <string>Details:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="details">
     <property name="text">
      <string comment="KDE::DoNotExtract">Kx = DH, Auth = RSA, MAC = SHA1</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sslVersionTag">
     <property name="text">
      <string>SSL version:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="sslVersion">
     <property name="text">
      <string comment="KDE::DoNotExtract">ElboniaTLS v0.0.0</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="certSelectorTag">
     <property name="text">
      <string>Certificate chain:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="certSelector"/>
   </item>
   <item row="6" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>239</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QTabWidget" name="certParties"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="trustedTag">
     <property name="text">
      <string>Trusted:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="trusted">
     <property name="text">
      <string comment="KDE::DoNotExtract">Maybe... no.</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="validityPeriodTag">
     <property name="text">
      <string>Validity period:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLabel" name="validityPeriod">
     <property name="text">
      <string comment="KDE::DoNotExtract">August 34 2004 to Undecimber 0 2008</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="serialTag">
     <property name="text">
      <string>Serial number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="serial">
     <property name="text">
      <string comment="KDE::DoNotExtract">23</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="digestTag">
     <property name="text">
      <string>MD5 digest:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="digest">
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="sha1DigestTag">
     <property name="text">
      <string>SHA1 digest:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="sha1Digest">
     <property name="text">
      <string comment="KDE::DoNotExtract">B4:DB:00:2E</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
